Code/GraphMol/MarvinParse/MarvinDefs.h

```diff
@@ -212,13 +219,50 @@ class MarvinRectangle {
                                           MarvinRectangle &r2);
 };
 
+template <typename T>
+typename std::vector<std::unique_ptr<T>>::iterator findUniquePtr(
+    std::vector<std ::unique_ptr<T>> &vector, T *itemToFind) {
+  auto foundUniqIter = find_if(vector.begin(), vector.end(),
+                               [itemToFind](std::unique_ptr<T> &uniquePtr) {
+                                 return uniquePtr.get() == itemToFind;
+                               });
+
+  if (foundUniqIter == vector.end()) {
+    throw FileParseException("Unexpected error - item to find not found");
+  }
+
+  return foundUniqIter;
+}
+
+template <typename T>
+void eraseUniquePtr(std::vector<std ::unique_ptr<T>> &vector, T *itemToErase) {
+  // auto removeUniqIter = find_if(vector.begin(), vector.end(),
```

Comment:
Do you want to keep that commented code?

Reply:
no